languages/julia/concepts/functions-introduction/links.json

```diff
@@ -1 +1,10 @@
-[]
+[
+    {
+        "url": "https://docs.julialang.org/en/v1/manual/functions/",
+        "description": "functions"
+    },
+    {
+        "url": "https://github.com/invenia/BlueStyle",
+        "description": "blue-style"
+    }
+]
```

Comment:
@ErikSchierboom should this file list _all_ links? Those aren't really standalone resources, they're just linked to give context.

Reply:
See the discussion on Slack :)

Reply:
This wasn't trivial to follow. The answer in #2499 seems to be "No, not all links".